docs/guide/catalog/set_up_products.md

```diff
@@ -0,0 +1,52 @@
+# Set up product for purchasing
+
+To enable purchasing from the catalog, the following configuration is required:
+
+- at least [one region and one currency for the shop](#region-and-currency)
+- [VAT rates for the product type](#vat-rates)
+- at least [one price for the product](#product-price)
+- [availability with positive or infinite stock for the product](#product-availability)
+
+## Region and currency
+
+All currencies available in the system must be enabled in the Back Office under **Commerce** -> **Currencies**.
+
+Additionally, you must configure currencies valid for specific SiteAccesses in configuration:
+
+``` yaml
+ibexa:
+    system:
+        shop:
+            product_catalog:
+                currencies:
+                    - EUR
+                    - USD
+                regions:
+                    - france
+                    - germany
+                    - norway
+```
+
+### Configuring other regions and currencies
+
+By default, the system always uses the first currency and the first region configured.
+
+To implement a different logic, for example a switcher for preferred currencies and regions,
+you need to use `CurrencyResolverInterface` and `RegionResolverInterface` in you customization.
```

Comment:
```suggestion
To implement a different logic, for example a switcher for preferred currencies and regions,
you need to use `CurrencyResolverInterface` and `RegionResolverInterface` in you customization.
```

`\Ibexa\Contracts\ProductCatalog\Events\CurrencyResolveEvent` and `\Ibexa\Contracts\ProductCatalog\Events\RegionResolveEvent` should be subscribed in order to customize this logic. 

Mentioned services are meet to be used to read current currency / region.